load.php

```diff
@@ -12,6 +12,14 @@
 	'accepted_args' => 1,
 );
 
+// Load the Cavalcade Runner CloudWatch extension.
+// This is loaded on the Cavalcade-Runner, not WordPress, crazy I know.
+if ( get_config()['cavalcade'] && class_exists( 'HM\\Cavalcade\\Runner\\Runner' ) ) {
+	// Load the common AWS SDK. bootstrap() is not called in this context.
+	require __DIR__ . '/lib/aws-sdk/aws-autoloader.php';
+	require_once( __DIR__ . '/lib/cavalcade-runner-to-cloudwatch/plugin.php' );
```

Comment:
Let's move these lines to a `bootstrap_cavalcade()` function in case we want to expand in future.

Also, this `require` should be consistent with `()`, and with `_once` (not needed)